modules/ROOT/content-nav.adoc

```diff
@@ -5,10 +5,13 @@ Generic Start
 
 * Introduction
 ** xref:index.adoc[About Aura]
+** xref:cloud-providers.adoc[Cloud provider marketplaces]
```

Comment:
I don't think that this belongs in the introduction. Let's move it to below _Quick start_.